source/documentation/getting_started/get_started.erb

```diff
@@ -16,6 +16,24 @@ Once you have a personal account, you can access the GOV.UK PaaS admin tool for
 
 Contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
 
+### Trial and paid accounts
+
+When your department, agency or team first [signs up to GOV.UK PaaS](https://www.cloud.service.gov.uk/signup), their account can be either a paid account, or a free trial account to try GOV.UK PaaS.
+
+With a trial account, your department, agency or team can:
+
+- manage [users](/orgs_spaces_users.html#users-and-user-roles)
+- create and manage [spaces](/orgs_spaces_users.html#spaces)
+- deploy and run [apps](/deploying_apps.html#deploying-apps)
+- use [backing service](/deploying_services) free plans
+
+However, your department, agency or team has a limited [quota](/managing_apps.html#quotas) of resources, and cannot:
+
+- set up [custom domains](/deploying_services/use_a_custom_domain/#managing-custom-domains-using-the-cdn-route-service)
```

Comment:
This has a dependency on https://www.pivotaltracker.com/story/show/169203174

Reply:
Merging PR after discussion with @lukemalchergds